Makefile

```diff
@@ -11,13 +9,8 @@ endif
 # Initialize BGO_SPACE
 export BGO_SPACE=$(shell pwd)
```

Comment:
is BGO_SPACE still needed?

Reply:
I am fine leaving it for now.  It does not impact anything and I have seen projects that do this same thing.